### PR TITLE
Make criterion optional in signature of register_default_init_args()

### PR DESCRIPTION
**Pull request by aleksei-kashapov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,8 +39,11 @@ model = resnet50()
 # Load a configuration file to specify compression
 nncf_config = NNCFConfig.from_json("resnet50_int8.json")
 
+train_loader = # Your dataloader
+loss_criterion = # Your criterion (optional)
```
Comment (vshampor) [inline README.md:43]: Shouldn't add these lines

**Review by vshampor — CHANGES_REQUESTED** [review]

**Comment by vshampor** [comment]
Also, capitalize the first word of the commit message

**Comment by vshampor** [comment]
Jenkins please retry a build

File: README.md (hunk 1 above)
Comment (aleksei-kashapov) [reply README.md:43]: okay. I will fix it in the following commits

**Review by vshampor — APPROVED** [review]

**Pull request merged**